sdk/eventhubs/azure-messaging-eventhubs-stress/src/main/resources/logback.xml

```diff
@@ -1,49 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-  <if condition='isDefined("DEBUG_SHARE")'>
-    <then>
-      <appender name="RollingFile" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${DEBUG_SHARE}/java-eventhubs-stress-test-${CONTAINER_NAME:-container}.log</file>
-        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-          <Pattern>%d %p [%t] %logger{100} %m%n</Pattern>
-        </encoder>
-        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
-          <!-- daily rollover and gzip logs -->
-          <fileNamePattern>${DEBUG_SHARE}/archived/java-eventhubs-stress-test-${CONTAINER_NAME:-container}-%d{yyyy-MM-dd}.log</fileNamePattern>
-        </rollingPolicy>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+    </encoder>
+  </appender>
+  <appender name="OpenTelemetry"
+            class="io.opentelemetry.instrumentation.logback.appender.v1_0.OpenTelemetryAppender">
+  </appender>
```

Comment:
again, for my learning, does this mean log back redirect all logging from SDK and stress tests to app-insight.?

Reply:
otel will consume logback logs and will forward them to appinsights.
it happened in appinsights agent before and now is configured explicitly

Reply:
Thank you for the reference